app/shared/components/Scripts/component.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const Scripts = () => {
+  return (
+    <div>
```

Comment:
React 16 allows for rendering without wrapping div, something like:

```
const ScriptItems = () => ([
  <script src='/ui/js/vendor/jquery.3.3.1.min.js'/>
  <script src='/ui/js/plugins/file-upload-shim.js'/>
]);
```

Reply:
Ahh fabulous - I didn't even realise we had upgraded it : )

Reply:
Addressed this - removed now